bc/web/static/css/dist/styles.css

```diff
@@ -1 +1,1533 @@
\ No newline at end of file
+/*
+! tailwindcss v3.2.7 | MIT License | https://tailwindcss.com
```

Comment:
Weird that this isn't minified?

Reply:
it should be minified, The `tailwind-reload` service is not creating a minified version, I will change the `npm` command to fix this.

Reply:
We only want this minimized in prod, right? In dev we might as well have the big file. I guess it's the same thing though until #110 is landed?